src/main/features/ccj/views/their-details.njk

```diff
@@ -55,7 +55,8 @@
               heading: 'Their address',
               name: undefined
             },
-            showCorrespondenceAddressSection = false
+            showCorrespondenceAddressSection = false,
+            postcodeLabel = 'Enter a postcode'
```

Comment:
I don't believe it matters for CCJ. Can you double check with art/martin that this change is required here?

Reply:
It's unused view. Removed!